main/templates/main/about.html

```diff
@@ -0,0 +1,58 @@
+{% extends "main/base.html" %}
+{% block content %}
+<div class="row">
+  <h1>About CHAMP</h1>
```

Comment:
Can you please add `class="ui header"` to headers.

Reply:
Classes added to headers.